src/button-group.test.visuals.ts

```diff
@@ -0,0 +1,169 @@
+import { expect, test } from '@playwright/test';
+
+test.describe('Button Group', () => {
+  test.describe('Light Mode', () => {
+    test.describe('orientation', () => {
+      test('horizontal', async ({ page }, test) => {
+        await page.goto('/iframe.html?id=button-group--button-group');
+        await page.waitForSelector('glide-core-button-group');
+        await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
+      });
+
+      test('vertical', async ({ page }, test) => {
+        await page.goto('/iframe.html?id=button-group--button-group');
+
+        const handle = await page.waitForSelector('glide-core-button-group');
+
+        await handle.evaluate((node) => {
+          node.orientation = 'vertical';
+          return node.updateComplete;
+        });
+
+        await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
+      });
+    });
+  });
+
+  test.describe('Dark Mode', () => {
+    test.describe('orientation', () => {
+      test('horizontal', async ({ page }, test) => {
+        await page.goto(
+          '/iframe.html?id=button-group--button-group&globals=theme:dark',
+        );
+
+        await page.waitForSelector('glide-core-button-group');
+        await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
+      });
+
+      test('vertical', async ({ page }, test) => {
+        await page.goto(
+          '/iframe.html?id=button-group--button-group&globals=theme:dark',
+        );
+
+        const handle = await page.waitForSelector('glide-core-button-group');
+
+        await handle.evaluate((node) => {
+          node.orientation = 'vertical';
+          return node.updateComplete;
+        });
+
+        await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
+      });
+    });
+  });
+});
+
+test.describe('With Icons', () => {
+  test.describe('Light Mode', () => {
+    test.describe('horizontal', () => {
+      test('', async ({ page }, test) => {
```

Comment:
Not sure what to name this test. Any ideas? `''`, `'default'`, `'no-variant'`, `'variantless'`, ?

Reply:
Another solution is to change the tests around and test `orientation` inside `variant`:

```ts
test.describe('With Icons', () => {
  test.describe('Light Mode', () => {
    test.describe('variant', () => {
      test('horizontal');
      test('vertical');
    });
  });
});
```

Or:

```ts
test.describe('With Icons', () => {
  test.describe('Light Mode', () => {
    test.describe('variant', () => {
      test.describe('icon-only', () => {
        test('horizontal');
        test('vertical');
      });
    });
  });
});
```